src/ol/format/filter.js

```diff
@@ -28,9 +27,9 @@ const _ol_format_filter_ = {};
  * @returns {!ol.format.filter.And} `<And>` operator.
  * @api
  */
-_ol_format_filter_.and = function(conditions) {
+export const and = function(conditions) {
```

Comment:
could you change it to `export function and(conditions) {`, like it's done elsewhere?

Reply:
Of course. Will do later.